packages/sdk38/src/lcdapi/staking.spec.ts

```diff
@@ -0,0 +1,508 @@
+import { assert, sleep } from "@cosmjs/utils";
+
+import { coin, coins } from "../coins";
+import { isPostTxFailure } from "../cosmosclient";
+import { makeSignBytes } from "../encoding";
+import { MsgDelegate, MsgUndelegate } from "../msgs";
+import { SigningCosmosClient } from "../signingcosmosclient";
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Why is this here in the middle of the file?

Reply:
Ah sorry, this is from my VSCode setup: tab completion will automatically add an import statement, but has no respect for eslint.